agent/bench-scripts/unittests

```diff
@@ -181,14 +181,16 @@ declare -A cmds=(
     [test-16]="pbench-uperf --help"
     [test-17]="pbench-fio --sysinfo=bad"
     [test-18]="pbench-fio --help"
-
+    [test-19]="pbench-fio --pre-iteration-script=foo"
```

Comment:
@ndokos you forgot to add the gold file which caused the unit tests failure.

Reply:
@chaitanyaenr: right you are. Added it.